manticore/__main__.py

```diff
@@ -58,6 +62,9 @@ def parse_arguments():
                               "(default mcore_?????)"))
     parser.add_argument('--version', action='version', version='Manticore 0.1.5',
                          help='Show program version information')
+    parser.add_argument('--tx', type=check_positive,
```

Comment:
`--tx` is an ambiguous name for this parameter; on first glance it looks like it represents data to transmit. Maybe `--max_transactions`, `--max_tx_count`, `--max_tx`, or `--tx_limit`?

Reply:
great point, for some reason i have a mild dislike of underscores in cli flags like that though lol
`--maxtx` `--txlimit` `--txmax`?

Reply:
Out of those, I like `--txlimit`